pom.xml

```diff
@@ -1665,7 +1677,11 @@
         <artifactId>plexus-classworlds</artifactId>
         <version>${plexus-classworlds.version}</version>
       </dependency>
-
+      <dependency>
+        <groupId>org.jboss.logging</groupId>
+        <artifactId>jboss-logging</artifactId>
+        <version>3.5.0.Final</version> <!-- Or choose 3.3.1.Final if more compatible -->
```

Comment:
Please clean up after *GPT. Assuming we have decided either `3.5.0.Final` or `3.3.1.Final`, either remove or update the comment to avoide future confusion.